alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php

```diff
@@ -930,7 +930,7 @@ private function serveVodEntry()
 				if ($event)
 				{
 					$this->entryId = $event->getSourceEntryId();
-					$sourceEntry = BaseentryPeer::retrieveByPK($this->entryId);
+					$sourceEntry = kSimuliveUtils::getSourceEntry($event);
```

Comment:
kSimuliveUtils::getSourceEntry will be merged from https://github.com/kaltura/server/pull/9970